modules/client-configuration/pages/clients-almalinux.adoc

```diff
@@ -0,0 +1,138 @@
+
+ifeval::[{suma-content} == true]
+
+The products you need for this procedure are:
+
+[[almalinux-channels-wizard]]
+[cols="1,1", options="header"]
+.CentOS Products - WebUI
+|===
+| OS Version | Product Name
+| {almalinux} 8 | {almalinux} 8 x86_64
+|===
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[almalinux-channels-suma-cli]]
+[cols="1,1", options="header"]
+.CentOS Channels - CLI
+|===
+| OS Version | Base Channel
+| {almalinux} 8 | almalinux8-x86_64
+|===
+
+include::snippets/addchannels_vendor_cli.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+The channels you need for this procedure are:
+
+[[almalinux-channels-uyuni-cli]]
+[cols="1,1,1,1", options="header"]
+.CentOS Channels - CLI
+|===
+| OS Version | Base Channel | Client Channel       | AppStream Channel
+| {almalinux} 8 | almalinux8      | almalinux8-uyuni-client | almalinux8-appstream
+|===
+
+include::snippets/addchannels_novendor_cli_multiarch.adoc[]
+
+[NOTE]
+====
+The client tools channel provided by [command]``spacewalk-common-channels`` is sourced from {uyuni} and not from {suse}.
+====
+
+endif::[]
+
+
+[IMPORTANT]
+====
+For {almalinux} 8 clients, add both the Base and AppStream channels.
+You require packages from both channels.
+If you do not add both channels, you cannot create the bootstrap repository, due to missing packages.
+====
+
+
+[NOTE]
+====
+You might notice some disparity in the number of packages available in the AppStream channel between upstream and the {productname} channel.
+You might also see different numbers if you compare the same channel added at a different point in time.
+This is due to the way that {almalinux} manages their repositories.
+{almalinux} removes older version of packages when a new version is released, while {productname} keeps all of them, regardless of age.
+====
+
+
+include::snippets/appstream_admon.adoc[]
+
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+endif::[]
+
+
+include::snippets/check_sync_cli.adoc[]
+
+
+
+== Create an Activation Key
+
+You need to create an activation key that is associated with your {almalinux} channels.
+
+For more information on activation keys, see xref:client-configuration:activation-keys.adoc[].
+
+
+
+ifeval::[{uyuni-content} == true]
+
+== Trust GPG Keys on Clients
+
+include::snippets/trust_gpg.adoc[]
+
+endif::[]
+
+
+
+== Register Clients
+
+{almalinux} clients are registered in the same way as all other clients.
+For more information, see xref:client-configuration:registration-overview.adoc[].
+
+
+== Manage Errata
+
+When you update {almalinux} clients, the packages already include include metadata about the updates.
```

Comment:
```suggestion
When you update {almalinux} clients, the packages include metadata about the updates.
```

Feels like there's a sentence missing here ... ?

Reply:
Yes, still a WIP :-D

Maybe I even have it locally but didn't commit yet. I remember I wrote something like "Therefore you do not need any special procedure to retrieve errata information"

Why this? Because AlmaLinux8 is a RHEL/CentOS8 clone, and for CentOS8 you have a special procedure.

Or I can remove the whole "Manage Errata"  section for AlmaLinux8. Just let me know.